dotnet/build-packages-dev.sh

```diff
@@ -0,0 +1,42 @@
+DEVELOPMENT_VERSION="0.2.114-prerelease"
```

Comment:
NIt - if the build part is always replaced automatically and never commited, we could maybe replace is by a `x` and update the regex in the action